usr/src/common/zfs/zfeature_common.c

```diff
@@ -210,8 +212,8 @@ zpool_feature_init(void)
 	    ZFEATURE_FLAG_READONLY_COMPAT, bookmarks_deps);
 
 	static const spa_feature_t filesystem_limits_deps[] = {
-	    SPA_FEATURE_EXTENSIBLE_DATASET,
-	    SPA_FEATURE_NONE
+		SPA_FEATURE_EXTENSIBLE_DATASET,
```

Comment:
Sorry, my bad. That one is correct. Instead, skein_deps & edonr_deps are wrong, as they are all spaces (no tabs) right now.